app/models/logstash_queries/date_range_top_n_field_query.rb

```diff
@@ -3,17 +3,18 @@ class DateRangeTopNFieldQuery < DateRangeTopNQuery
 
   def initialize(affiliate_name, start_date, end_date, filter_field, filter_value, agg_options = {})
     super(affiliate_name, start_date, end_date, agg_options)
-    @filters = { 'affiliate' => @affiliate_name, filter_field => filter_value }.compact
+    @filters = { 'params.affiliate' => @affiliate_name,
+                 filter_field => filter_value }.compact
```

Comment:
since you are already factoring this you could also replace the hash rockets. But I will leave it up to you since it was already that way.

Reply:
As `params.affiliate` is a string, and `filter_field` is a variable pointing to a string, the recommended style is to use hash rockets:
https://rubystyle.guide/#no-mixed-hash-syntaces

Reply:
It still can be done :)
 { "params.affiliate": @affiliate_name,
   "#{filter_filed}": filter_value }